src/stdbuf/stdbuf.rs

```diff
@@ -59,7 +59,7 @@ fn preload_strings() -> (&'static str, &'static str) {
 
 #[cfg(target_os = "macos")]
 fn preload_strings() -> (&'static str, &'static str) { 
-    ("DYLD_INSERT_LIBRARIES", ".dylib")
+    ("DYLD_LIBRARY_PATH", ".dylib")
```

Comment:
It works as expected on my MacBook. I consulted the Apple's `dyld` documentation to confirm:

https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/dyld.1.html